views/frenchpoetry.erb

```diff
@@ -0,0 +1,59 @@
+<html>
+  <head>
+      <title>A poem by Victor Hugo</title>
+
+  </head>
+  <header>
+    <h2>Demain, dès l’aube, a poem by Victor Hugo</h2>
+  </header>
+
+  <section>
```

Comment:
The header and h2 are good - I was referring to everything below the `section` on line 10